ci-operator/step-registry/cucushift/installer/rehearse/ibmcloud/ipi/disconnected/private/provision/cucushift-installer-rehearse-ibmcloud-ipi-disconnected-private-provision-chain.yaml

```diff
@@ -50,5 +51,11 @@ chain:
     default: "DEFAULT_ENDPOINT"
   - name: SERVICE_ENDPOINT_GlobalTagging
     default: "DEFAULT_ENDPOINT"
+  - name: SERVICE_ENDPOINT_COSConfig
```

Comment:
The install workflow is used for both  <4.18 and 4.18+ versions, for the new 4.18 supported endpoints, you should  overwrite them in job config level, but not provision chain, or you need to create another new install workflow yaml for 4.18+